src/modules/previewpane/SvgPreviewHandler/Settings.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.PowerToys.Settings.UI.Library;
+
+namespace SvgPreviewHandler
+{
+    internal sealed class Settings
+    {
+        private static readonly SettingsUtils ModuleSettings = new SettingsUtils();
+
+        public int ColorMode
+        {
+            get
+            {
+                try
+                {
+                    return ModuleSettings.GetSettings<PowerPreviewSettings>(PowerPreviewSettings.ModuleName).Properties.SvgBackgroundColorMode.Value;
+                }
+                catch (FileNotFoundException)
```

Comment:
is this the only exception that can happen here?

Reply:
I checked other modules for reference, and they too only catch this exception.